litmus-portal/frontend/src/containers/app/App-styles.ts

```diff
@@ -2,7 +2,7 @@ import { makeStyles, Theme } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
-    width: '100vw',
+    width: 'auto',
```

Comment:
If you keep auto width on root in case any of the components or layout takes up extra width the main frame would go out of range and give a horizontal scrollbar. Which is constrained with 100vw.

Reply:
I was facing the issue with `width: 100vw` the `horizontal` scroll was coming